### DISCUSSION TITLE
TSC Meeting August 2022

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#7402] [Team Posts]
@thesofproject/steering-committee Now that most members are back from summer holidays and v2.2 was released in early summer lets look forward and align on v2.3 features and v2.4 planning. 

Register your availability:  https://doodle.com/meeting/participate/id/dNxZqPpa

**Agenda** please add/edit items.

1) Zephyr update and status. Next steps.

2) IPC4 and Windows support. 

3) Old Intel platform deprecation. Intel plans to park some older HW at v2.2

4) RTOS partitioning. Partitioning the codebase for clean separation of rtos code from common code.

5) Tooling update - Integrating with Zephyr and Windows based tools.

6) Separate repos for tooling and topologies - Using "west" now permits alignment of multiple repos and means its easier to release updates for topologies and tools without a FW release.

7) Enroll SOF to become Open Source Security Foundation (OpenSSF) 
 compliant project like Zephyr - see Zephyr example https://bestpractices.coreinfrastructure.org/en/projects/74

**Comment by dbaluta** [comment]
@lgirdwood https://doodle.com/meeting/participate/id/dNxZqPpa

Let's see if we can meet next week.

**Comment by lgirdwood** [comment]
@thesofproject/amd @thesofproject/mediatek pls register your availability and suggest topics for discussion and alignment.

**Comment by lgirdwood** [comment]
@thesofproject/amd @thesofproject/google @thesofproject/intel @thesofproject/mediatek @thesofproject/nxp Thursday 3 - 4PM GMT is so far looking like the best time. Will send invite out tomorrow AM UK time to give time for final topics and availability.

**Comment by bhiregoudar** [comment]
> @thesofproject/amd @thesofproject/mediatek pls register your availability and suggest topics for discussion and alignment.

Thanks, Done from AMD

**Comment by kuanhsuncheng** [comment]
Thanks,  I will inform our menbers of date. (MediaTek)

**Comment by bhiregoudar** [comment]
> @thesofproject/amd @thesofproject/google @thesofproject/intel @thesofproject/mediatek @thesofproject/nxp Thursday 3 - 4PM GMT is so far looking like the best time. Will send invite out tomorrow AM UK time to give time for final topics and availability.

Didn't receive the meeting invite, meeting is there today?

**Comment by dbaluta** [comment]
@bhiregoudar what is your email address i can forward it to you

**Comment by kuanhsuncheng** [comment]
HI Daniel, @dbaluta 

We didn't receive the meeting invite either.

**Comment by dbaluta** [comment]
> HI Daniel, @dbaluta
> 
> We didn't receive the meeting invite either.

@kuanhsuncheng please check email now. We need to sync better for the next meetings :)

**Comment by dbaluta** [comment]
Here how to meeting shows in my calendar. Note that time is in Eastern European Time zone.
Meeting link: [meet.google.com/jau-fpca-ohq](https://meet.google.com/jau-fpca-ohq?hs=224)


![image](https://user-images.githubusercontent.com/62316/186644458-cb0eea73-9dbd-41b9-8119-20f770c6c1bd.png)

**Comment by bhiregoudar** [comment]
> @bhiregoudar what is your email address i can forward it to you

basavaraj.hiregooudar@amd.com

**Comment by dbaluta** [comment]
> @bhiregoudar what is your email address i can forward it to you
> 
> basavaraj.hiregooudar@amd.com

You sure this is ok? 

![image](https://user-images.githubusercontent.com/62316/186651725-f6c01ec0-04a7-4578-a7e2-967ab40d6b4e.png)

**Comment by bhiregoudar** [comment]
> @bhiregoudar what is your email address i can forward it to you
> 
> basavaraj.hiregooudar@amd.com
> 
> You sure this is ok?

sorry, it is basavaraj.hiregoudar@amd.com

**Comment by kuanhsuncheng** [comment]
@dbaluta,

We see the  meeting invite, thanks for your help.

**Comment by lgirdwood** [comment]
Thanks @dbaluta - we need to gather a list of emails for next time. GH usernames don't work for meetings !
